src/pages/index.js

```diff
@@ -103,12 +136,13 @@ profileEditButton.addEventListener("click", () => {
   editProfilePopup.open();
 });
 
-//add card popup
+/* ---------------------------- add card popup ---------------------------- */
 
 const handleAddCardFormSubmit = (inputValues) => {
   const card = renderCard(inputValues);
   section.addItem(card);
   newCardPopup.close();
```

Comment:
Needs correction: you should add the card on the server andonly after the successful api response update the cards on the page and close the popup